Cargo.toml

```diff
@@ -32,6 +32,11 @@ thiserror = "1.0.30"
 lazy_static = "1.4.0"
 tracing = "0.1.37"
 tracing-subscriber = "0.3.17"
+opentelemetry-appender-tracing = { version = "0.27.0", default-features = false}
+opentelemetry = { version = "0.27.0", default-features = false, features = ["trace"] }
+opentelemetry_sdk = { version = "0.27.0", default-features = false, features = ["trace", "rt-tokio"] }
+opentelemetry-http = { version = "0.27.0", default-features = false}
+opentelemetry-otlp = { version = "0.27.0", features = ["logs", "http-json", "reqwest-client"] }
```

Comment:
note: I used the `http+json` version of opentelemetry instead of the grpc+protobuf

I did this because we currently only support the `http+json` approach in Paima at the moment, but plan to support grpc+protobuf in the future